lib/token.js

```diff
@@ -343,6 +343,10 @@ function getIdToken(sdk, oauthOptions, options) {
     throw new AuthSdkError('A clientId must be specified in the OktaAuth constructor to get an idToken');
   }
 
+  if (util.isString(oauthParams.responseType) && oauthParams.responseType.indexOf(' ') !== -1) {
```

Comment:
It should be. The way the authorize endpoint works, it accepts a `responseType` that's space separated ie. `id_token token`. We don't want devs to pass in these response types as a string and space-separation seems like the most common slipup.

Also, space-separation is the only scenario where the api still works, but the sdk doesn't (it'd simply return the first token in the list). If a user passed a comma, semicolon, or anything else, the api would throw an error and the sdk would pass that error to the dev.